src/pandablocks_ioc/_pvi.py

```diff
@@ -61,9 +63,16 @@ def add_pvi_info(
     useComboBox: bool = record_creation_func == builder.mbbOut
 
     if record_creation_func == builder.Action:
-        # TODO: What value do I write? PandA uses an empty string
-        component = SignalX(record_name, record_name, value="")
-        access = "x"
+        if record_name == "PCAP:ARM":
+            component = SignalRW(
+                record_name, record_name, widget=ButtonPanel(), read_widget=LED()
+            )
+            access = "rw"
+
+        else:
+            # TODO: What value do I write? PandA uses an empty string
```

Comment:
Can you answer this comment? I think I wrote it as I know very little about PVI and how it would send values across - you likely know more than me at this point. If nothing else, just remove it.